.tool-versions

```diff
@@ -1 +1,2 @@
 nodejs 22.14.0
+yarn 1.22.22
```

Comment:
@damassi I had to specify the yarn version in order for the app to work now that we upgraded node. 🤷